spring-5-security/src/main/java/com/baeldung/xss/XSSUtils.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.xss;
+
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+import org.owasp.esapi.ESAPI;
+
+public class XSSUtils {
+    private XSSUtils() {
+    }
+
+    public static String stripXSS(String value) {
+        if (value != null) {
```

Comment:
switch to `if (value == null) { return null; }` and stop redefining `value` throughout this function.

Reply:
Done.